packages/validator/test/cli-validator/mock-files/oas3/test-formats.yaml

```diff
@@ -0,0 +1,66 @@
+openapi: 3.0.0
```

Comment:
This new testcase should make it easy to test changes to this rule in the future.
Previously, the only test we had for the invalid_type_format_pair rule was tied up with the oneOf testing.